cmd/minikube/cmd/dashboard.go

```diff
@@ -65,19 +65,30 @@ var dashboardCmd = &cobra.Command{
 		}
 		cluster.EnsureMinikubeRunningOrExit(api, 1)
 
+		fmt.Fprintln(os.Stderr, "Enabling dashboard ...")
+		// Enable the dashboard add-on
+		err = configcmd.Set("dashboard", "true")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Unable to enable dashboard: %v\n", err)
+			os.Exit(1)
+		}
+
 		ns := "kube-system"
 		svc := "kubernetes-dashboard"
-		if err = util.RetryAfter(30, func() error { return service.CheckService(ns, svc) }, 1*time.Second); err != nil {
+		fmt.Fprintln(os.Stderr, "Verifying dashboard health ...")
+		if err = util.RetryAfter(180, func() error { return service.CheckService(ns, svc) }, 1*time.Second); err != nil {
```

Comment:
It would be interesting to keep a "watch" on the dashboard pod + on a separate line the result of the service status? But this idea might be better off in a separate PR. It's just 3 minutes of potential waiting is definitely long. Out of those 1 minute is almost guaranteed to be just the addon-manager kicking in...

Reply:
A good idea, but due to the complexity l will leave this for another PR. This step generally only takes a second or two, but to avoid flakes in places with poor connectivity, I wanted to make sure to wait an extended period for the pod to come up.